### DISCUSSION TITLE
Cannot read properties of undefined (reading 'init')

### DISCUSSION THREAD

**Discussion by eddiehe99** [walinejs/waline#992] [Q&A] [answered]
使用：
import Waline from '@waline/client';
const waline = Waline.init({ /* options */ })
会报 Cannot read properties of undefined (reading 'init') 的错误，

改成使用：
import * as Waline from '@waline/client';
const waline = Waline.init({ /* options */ })

或者改成使用：
import { init } from '@waline/client';
const waline = init({ /* options */ })

都会报 [Cannot destructure property 'comment' of 'undefined' as it is undefined.](https://github.com/walinejs/waline/discussions/991) 的错误。

**Comment by Mister-Hope** [comment, marked as answer]
Waline has no default export

```ts
import { init } from '@waline/client';

const instance = init({/* options */});
```

**Reply by eddiehe99** [reply]
会报 [Cannot destructure property 'comment' of 'undefined' as it is undefined.](https://github.com/walinejs/waline/discussions/991) 的错误。

**Reply by Mister-Hope** [reply]
Minimal reproduction repo plz

**Reply by eddiehe99** [reply]
Uncaught (in promise) TypeError: Cannot destructure property 'comment' of 'undefined' as it is undefined.
init.ts?d4e7:95

The error occurs when using waline.update( ).

**Comment by Mister-Hope** [comment]
Fixed in 2.2.0

**Reply by eddiehe99** [reply]
2.2.0 works well.
Thanks!

**Comment by Mister-Hope** [comment]
Fixed in 2.2.0
